test/client_test.py

```diff
@@ -73,7 +73,7 @@ def test_state_event():
 
     room.name = False
     room.topic = False
-    room.aliases = False
+    room.aliases = []
```

Comment:
I found that when initializing aliases in rooms.py used empty list, so using `False` here doesn't look correct.